### PR TITLE
Mic and Camera device selection modal window for Personal Training 

### PR DESCRIPTION
**Pull request by retif**
PRODDEV-322

## Steps to test:

- [ ] Create a personal training
- [ ] Open personal training page
- [ ] Click "Join Meeting"
- [ ] Click "Settings
- [ ] Test any of the available input devices.

### CODE REVIEW

**Comment by hamrant** [comment]
retest

File: js/gated-content/src/components/personal-training/modal/deviceManager.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        >
+          <option
+            v-for="device in audioInputDevices"
+            :key="device.deviceId"
+            :value="device.deviceId"
+          >{{ device.label }}
+          </option>
+        </select>
+      </div>
+      <!--div
```
Comment (hamrant) [inline js/gated-content/src/components/personal-training/modal/deviceManager.vue:24]: Can we remove commented code?

**Comment by anpolimus** [comment]
retest

**Comment by anpolimus** [comment]
@retif please, delete commented code and we are ready to go

**Comment by AnastasiiaPys** [comment]
Tested. Works good!

**Review by AnastasiiaPys — APPROVED** [review]

**Comment by AnastasiiaPys** [comment]
Retested. Works as expected.

**Pull request merged**